### ISSUE TITLE
[0.1.6] NSME Crash generating map caused by moonlight.api

### ISSUE THREAD

**Issue by leagris** [AllTheMods/ATM-9#429]
ATM9 Version: 0.1.6

Generating a new vanilla map triggers a No Such Method Error exception during call to non-existing method: `net.mehvahdjukaar.moonlight.api.util.math.colors.LABColor`:

[crash-2023-09-22_19.01.37-client.txt](https://gist.github.com/leagris/fb748679df581a4f60a4fdecff7f98f6)
```none
java.lang.NoSuchMethodError: 'net.mehvahdjukaar.moonlight.api.util.math.colors.LABColor net.mehvahdjukaar.moonlight.api.util.math.colors.LABColor.multiply(float, float, float, float)'
	at net.mehvahdjukaar.supplementaries.common.misc.ColoredMapHandler$ColorData.processTexture(ColoredMapHandler.java:257) ~[supplementaries-1.20-2.6.2.jar%23790!/:?] {re:mixin,re:classloading}
```

**Comment by leagris** [comment]
FIX Available:

Crash is solved by updating Moonlight API to 2.8.28
https://www.curseforge.com/minecraft/mc-mods/selene/files/4761723

**Comment by leagris** [comment]
Resolved in 0.1.7
